packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -1,12 +1,12 @@
-import { Button, Drawer, Icon, Popover, Tabs } from '@deriv/components';
+import { Button, Drawer, Icon, Popover, Tabs, Modal, Dialog } from '@deriv/components';
```

Comment:
Do we use `Dialog` anywhere?

Reply:
> Do we use `Dialog` anywhere?

Yes we have a Dialog in code but I've checked and this dialog got from bot-web-ui 

please check following URL line 88:
https://github.com/binary-com/deriv-app/blob/dev/packages/bot-web-ui/src/components/run-panel.jsx

Reply:
> > Do we use `Dialog` anywhere?
> 
> Yes we have a Dialog in code but I've checked and this dialog got from bot-web-ui
> 
> please check following URL line 88:
> https://github.com/binary-com/deriv-app/blob/dev/packages/bot-web-ui/src/components/run-panel.jsx

Done